### DISCUSSION TITLE
PushNotifications <-> azure notifications hub support

### DISCUSSION THREAD

**Discussion by baaaaif** [microsoft/WindowsAppSDK#2832] [Q&A]
I'm in the process of migrating an existing push functionality from a UWP app to the WASDK. 
We're sending messages through azure notification hubs and I'm wondering if azure notification hubs support WASDK?
Currently the package SID is entered in azure. Does this still work, or do I have to wait for a possibility with azure app id and tenant id + secret?
Unfortunately, I can't find anything about this in the documentation... not in the WASDK and not in the azure notification hub documentation

**Comment by baaaaif** [comment]
ok, seems like nobody can help right now.
Does anyone on the dev team have a suggestion as to the right place to ask this?

@gabbybilka can you help?

**Reply by gabbybilka** [reply]
I'm assuming you've taken a look at this documentation: https://learn.microsoft.com/windows/apps/windows-app-sdk/notifications/push-notifications/push-quickstart

@loneursid can you weigh in?

**Reply by baaaaif** [reply]
Yes i did and followed the instructions. pfn-mapping is also created.
Now i need to know how to migrate from UWP Push-Partner-Center SID-Configuration in our notification hub
[https://learn.microsoft.com/en-us/azure/notification-hubs/configure-notification-hub-portal-pns-settings?tabs=azure-portal#windows-push-notification-service](https://learn.microsoft.com/en-us/azure/notification-hubs/configure-notification-hub-portal-pns-settings?tabs=azure-portal#windows-push-notification-service)

to an Windows App  SDK equivalent, which doesn't support MS Partner Center.
From your quickstart link: "Note: We do NOT support using Windows App SDK push notifications with Microsoft Partner Center"

**Reply by koystinen** [reply]
@baaaaif have you ever figured out how to do this migration? and get WindowsAppSDK work with Azure Notification Hubs?

**Reply by baaaaif** [reply]
unfortunately no

**Reply by koystinen** [reply]
@gabbybilka and @loneursid Would you be able to provide any guidance? Our team is facing similar challenge to figure out if it possible to make Azure Notification Hub send notification via WNS to app build with WindowsAppSDK (WinUI)

**Reply by gabbybilka** [reply]
As I am no longer working on this team, @RDMacLachlan can you help and redirect to the proper folks? Thanks.

**Reply by koystinen** [reply]
Thank you! @gabbybilka 
@RDMacLachlan Would you be able to assist?

**Comment by minze-it** [comment]
Hey @gabbybilka , do you know when push functionality will be supported by Windows App SDK Apps with Azure Notification Hub?

**Comment by ekalchev** [comment]
@michael-hawker Can anyone from Microsoft answer this question?
